### PR TITLE
OPNET-133: Enabling remote worker feature only for baremetal platform

### PR DESCRIPTION
**Pull request by tsorya**

### CODE REVIEW

File: cmd/runtimecfg/node-ip.go [hunk 1]
```diff
@@ -269,3 +270,8 @@ func parseIPs(args []string) ([]net.IP, error) {
 	}
 	return ips, nil
 }
+
+// currently we allow setting remote worker only in case of baremetal platform without external lb
+func setRemoteWorker(vips []net.IP, matchesVips bool) bool {
```
Comment (mkowalski) [inline cmd/runtimecfg/node-ip.go:275]: Can we rename from `set` to `is`? This function as I see it is only answering a question and is not setting any state anywhere

**Comment by cybertron** [comment]
/lgtm

I don't see anything ip version specific here so as long as the required ipv6 job passes we should be good.

**Pull request merged**